packages/cli/src/utils/txevent.ts

```diff
@@ -0,0 +1,13 @@
+import { TxResult } from "@mesg/api/lib/lcd";
+
+const isAction = (action: string) => (x: any) => x.key === 'action' && x.value === action
+const isModule = (module: string) => (x: any) => x.key === 'module' && x.value === module
+const isHash = (x: any) => x.key === 'hash'
+
+export const findHash = (txResult: TxResult, resourceType: 'service', action: string): string => {
```

Comment:
why put service as defaut here?
```suggestion
export const findHash = (txResult: TxResult, resourceType: string, action: string): string => {
```

Reply:
This is not a default, this is the type so you can only pass `"service"` as value otherwise typescript will throw an error.